.github/workflows/verify.yaml

```diff
@@ -32,9 +32,14 @@ jobs:
         working-directory: go/src/github.com/shipwright-io/build
         args: --timeout=10m
     - name: Install Counterfeiter
+      env:
+        GOPATH: ${GITHUB_WORKSPACE}/go
       run: |
-        go install github.com/maxbrunsfeld/counterfeiter/v6@latest
+        go version
+        make -C $GOPATH/src/github.com/shipwright-io/build install-counterfeiter
     - name: Run verify-generate
+      env:
+        GOPATH: ${GITHUB_WORKSPACE}/go
       run: |
-        export GOPATH="${GITHUB_WORKSPACE}"/go
+        #export GOPATH="${GITHUB_WORKSPACE}"/go
```

Comment:
```suggestion
```

Reply:
You don't need to review the changes in my DEBUG commit. That will go away.